.github/workflows/app-react.yml

```diff
@@ -55,65 +50,3 @@ jobs:
           env_vars: javascript
           # Specify whether or not CI build should fail if Codecov runs into an error during upload
           fail_ci_if_error: true
-
-      - name: SonarQube Scan
```

Comment:
Removed - The results of this scan are used to generate a comment in the pull request thread.